app/src/main/java/com/ls/ui/fragment/LocationFragment.java

```diff
@@ -110,23 +118,36 @@ private void fillMapViews(List<Location> locations) {
         uiSettings.setMyLocationButtonEnabled(false);
         uiSettings.setCompassEnabled(false);
         uiSettings.setRotateGesturesEnabled(false);
+
+        mGoogleMap.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
+
+            @Override
+            public boolean onMarkerClick(Marker marker) {
```

Comment:
@iamareebjamal @Shailesh351 @the-dagger @niranjan94 Can you see, please? It's been a while.